src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,12 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns true if the command mutates AddressBook data.
+     * @return
```

Comment:
Rather than use an empty `@return/@param/@throws`, either give a description as well, or delete it altogether.